core/src/mindustry/entities/abilities/ShieldArcAbility.java

```diff
@@ -79,6 +82,14 @@ public void addStats(Table t){
 
     @Override
     public void update(Unit unit){
+        if(instanceFull) {
+            if (data < 0 && !broken) broken = true;
```

Comment:
Incorrect formatting in several places (extra space), see CONTRIBUTING.md.

Reply:
OMG, I'm very sorry, I didn't check it carefully. Do I need to delete the excess parts and resend it again?